lib/modules/ssh/strong_password/__init__.py

```diff
@@ -1,7 +1,35 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
+import time
 from core.compatible import generate_token
+from database.connector import insert_honeypot_events_from_module
+from os import path
+
+
+class ModuleProcessor:
+    """
+    this is the processor to run after docker machine is up to grab the log files or do other needed process...
+    """
+    def __init__(self):
+        self.kill_flag = False
+        self.pointer=''
+
+    def processor(self):
+        """
+        processor function will be called as a new thread and will be die when kill_flag is True
+        :return:
+        """
+        open('tmp/logins.txt', 'w').close() #empty the previous logs if any
+        while not self.kill_flag:
+            if(path.exists('tmp/logins.txt')):
+                f = open("tmp/logins.txt", "r")
+                file_list=list(f)
+                if self.pointer!=file_list[-1:] and file_list:
+                    self.pointer=file_list[-1:]
+                    username,password,ip,module_name = self.pointer[0].split(":")
```

Comment:
if you use json as i mention, you'll prase it easier,

imagin a password has `:` in it.

`root:pass:word:1.1.1.1,ssh/strong_password`

then its an error...

and add the information i've mentioned...

Reply:
sure ,I did not think of this.